Source/UI/NewUI/PaymentForm/Factory/PaymentFormFactory.swift

```diff
@@ -1,27 +1,29 @@
 import UIKit
+import Checkout
 
 public struct PaymentFormFactory {
+  public static func getPaymentFormViewController(environment: Environment = .live,
```

Comment:
Could we unit test this method to check the factory creates a ViewController with all the expected properties?